src/main/scala/is/hail/io/bgen/BgenRecord.scala

```diff
@@ -1,9 +1,12 @@
 package is.hail.io.bgen
 
+import com.google.common.io.LittleEndianDataInputStream
```

Comment:
I can't tell where this is used. Also, can you explain what the new Inflater, InflaterInputStream classes are doing?